CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+Change Log
+==========
+
+1.3
+---
+* Change FxATestAccount constructor to accept the url to the FxA API server
+  * This is a **breaking change**
+
+1.2
+---
+* Update required version of PyFxA in setup.py
+
+1.1
+---
+* Update required version of PyFxA in requirements.txt
```

Comment:
Oh, and for 1.1 and 1.2 I'd reword to 'Update required version of PyFxA to x.x' rather than mention the file that maintains the dependency.